modules/dmrpp_module/DmrppCommon.cc

```diff
@@ -546,6 +546,14 @@ DmrppCommon::print_chunks_element(XMLWriter &xml, const string &name_space)
             throw BESInternalError("Could not write fillValue attribute.", __FILE__, __LINE__);
     }
 
+    if(d_chunks.empty()){
+        if(d_byte_order.empty()==false) {
```

Comment:
The d_chunks.empty() and d_byte_order.empty() == false tests could be combined. It's tempting to combine all three, but since the third 'if' is run for side effect, I think that would make it pretty hard to read.